src/app/component/Typography.jsx

```diff
@@ -26,6 +28,11 @@ const H5 = ({ children, ...rest }) => (
     {children}{" "}
   </Typography>
 );
+const H6 = ({ children, ...rest }) => (
+  <Typography variant="h6" {...rest}>
+    {children}{" "}
+  </Typography>
+);
 const Body1 = ({ children, ...rest }) => (
   <Typography variant="body1" {...rest}>
     {children}{" "}
```

Comment:
extra space

Reply:
It was there. Not sure why we have it. Do you think that we should remove it?

Reply:
yes, we should